easybuild/easyconfigs/f/FastANI/FastANI-1.1-memcpy.patch

```diff
@@ -0,0 +1,13 @@
+remove memcpy wrapper 
```

Comment:
@fizwit Sorry to be a bit pedantic here, but can you clarify *why* this is needed (by enhancing the comment in the patch file)?

Also mention yourself as an author of the patch, something like:

```
remove memcpy wrapper to fix problem with ...
author: John Dey (Fred Hutch)
```

Reply:
@fizwit Ping on this?

Reply:
I suspect the reason is similar to why we've done this for `Mash`? Cfr. https://github.com/easybuilders/easybuild-easyconfigs/pull/7226/commits/12eb6d952f6d2164a114e3e627a4bc99f0b176b5

Reply:
tackled this myself via #7992, since I had a need for `FastANI` built with `intel/2018b` anyway...